templates/quickstart-vpc-for-atlassian-services.yaml

```diff
@@ -14,6 +14,10 @@ Metadata:
           - PublicSubnet1CIDR
           - PublicSubnet2CIDR
           - VPCCIDR
+      - Label:
+          default: Bastion host provisioning
+        Parameters:
+          - BastionHostRequired
```

Comment:
I would include the Keypair in this group as it's only used for the Bastion host now